packages/common/src/index.ts

```diff
@@ -359,6 +360,58 @@ export default class Common {
     }
   }
 
+  /**
+   * Internal helper function to calculate a fork hash
+   * @param hardfork Hardfork name
+   * @returns Fork hash as hex string
+   */
+  _calcForkHash(hardfork: string) {
+    const genesis = Buffer.from(this.genesis().hash.substr(2), 'hex')
+
+    function pad(str: string, max: number): string {
+      return str.length < max ? pad('0' + str, max) : str
+    }
+
+    let hfBuffer = Buffer.alloc(0)
+    let prevBlock = 0
+    for (const hf of this.hardforks()) {
+      const block = hf.block
+
+      // Skip for chainstart (0), not applied HFs (null) and
+      // when already applied on same block number HFs
+      if (block !== 0 && block !== null && block !== prevBlock) {
+        const hfBlockBuffer = Buffer.from(pad(block.toString(16), 16), 'hex')
+        hfBuffer = Buffer.concat([hfBuffer, hfBlockBuffer])
+      }
+
+      if (hf.name === hardfork) break
+      prevBlock = block
+    }
+    const inputBuffer = Buffer.concat([genesis, hfBuffer])
+
+    // CRC32 delivers result as signed (negative) 32-bit integer,
+    // convert to hex string
+    const forkhash = new Number(crc32Buffer(inputBuffer) >>> 0).toString(16)
+    return `0x${forkhash}`
+  }
+
+  /**
+   * Returns an eth/64 compliant fork hash (EIP-2124)
+   * @param hardfork Hardfork name, optional if HF set
+   */
+  forkHash(hardfork?: string) {
+    hardfork = this._chooseHardfork(hardfork, false)
+    const data = this._getHardfork(hardfork)
+    if (data['block'] === null) {
+      let msg = 'No fork hash calculation possible for non-applied or future hardfork'
```

Comment:
nitpick pointed by my editor.
```suggestion
      const msg = 'No fork hash calculation possible for non-applied or future hardfork'
```

Reply:
Fixed.